lms/static/js/discussions_management/views/discussions.js

```diff
@@ -80,6 +87,42 @@
                     this.notification.render();
                 },
 
+                cohortStateUpdate: function(state) {
+                    if ($('.discussions-management').data('enrollment-track-count') <= 1) {
+                        this.showDiscussionManagement(state.is_cohorted);
+                    } if (this.getSelectedScheme() !== COHORT) {
+                        this.showCohortSchemeControl(state.is_cohorted);
+                    }
+                },
+
+                showDiscussionManagement: function(show) {
+                    if (!show) {
+                        $('.btn-link.discussions_management').addClass(HIDDEN_CLASS);
+                        $('#discussions_management').addClass(HIDDEN_CLASS);
+                    } else {
+                        $('.btn-link.discussions_management').removeClass(HIDDEN_CLASS);
+                        $('#discussions_management').removeClass(HIDDEN_CLASS);
+                    }
+                },
+
+                showCohortSchemeControl: function(show) {
+                    if (!show) {
+                        $('.division-scheme-item.cohort').addClass(HIDDEN_CLASS);
+                        this.updateSchemeSelectionLayout(2);
+                    } else {
+                        $('.division-scheme-item.cohort').removeClass(HIDDEN_CLASS);
+                        this.updateSchemeSelectionLayout(3);
```

Comment:
@staubina this logic isn't correct. It also needs to take into account whether or not the enrollment track is present.